recipes/mars/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "MARS" %}
+{% set version = "1.0" %}
+
+package:  
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+source:
+  url: https://github.com/maiziex/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: a4ba7660a68ccc6fffeaa9bc2c64c1bd5a46837e93fce7e3bc0a5710e8deb36c
+
+requirements:
+  host:
+    - python >=3
+    - setuptools
+
+  run:
+    - python >=3
+    - biopython
+    - samtools
+    - pysam
+    - minimap2
+    - openpyxl
+    - pandas
+
+test:
+  commands:
+    - MARS_step1 --help
+    - MARS_step2 --help
+
+
+about:
+  home: https://github.com/maiziex/MARS
+  license: MIT
+  summary: Multiple Alignment-based Refinement of SVs(MARS)
```

Comment:
Can you add `license_file` here as well please?

Could you also add yourself as a maintainer to this recipe?

Reply:
should I create a new pull request? Or how can I edit it based on this pr?

Reply:
No, please don't create a new PR. You can just push new commits to your branch.

Or if you aren't familiar with git you should be able to edit here: https://github.com/volcano1998/bioconda-recipes/edit/yourpackage3/recipes/mars/meta.yaml

Reply:
Hello Simon, I already added license file, thank you for your direction!  However, I'm not sure about "adding myself as a maintainer". Could you give me some guide if you don't mind? Thank you!

Reply:
Hi @volcano1998, I added it here: https://github.com/bioconda/bioconda-recipes/pull/30685/commits/3b865eb3f04fa4ee8e32d998bd5abca56fba7000